### PR TITLE
HEYUI-164: Add others icon

### PR DESCRIPTION
**Pull request by Hey-Komal**
## Pull Request

**Added icon** for `Others` required for https://heycar.atlassian.net/browse/FR-735

### Description

image

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation change

### How do I test this

- Add steps to test
- in bullet point format
- preferably you can add link to the storybook build in the PR

### Did you remember to take care of the following?

- [ ] `npm i` – for new NPM dependencies.
- [x] `npm run lint` - to check for linting issues
- [x] `npm run test` - to run unit tests
- [ ] `npm run test:sh:docker` - to run screenshot tests, [detail instruction](https://hey-car.github.io/heycar-uikit/main/?path=/docs/guidelines-screenshot-testing--page)

### New Feature / Bug Fix

- [ ] Run unit tests to ensure all existing tests are still passing.
- [ ] Add new passing unit tests to cover the code introduced by your pr.

Thanks for contributing!

### CODE REVIEW

**Comment by esraltintas** [comment]
@Hey-Komal Could you check the screenshot tests, please?

File: packages/icons/src/paths/Others.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import createSvgIcon from '../utils/createSvgIcon';
+
+export const Others = createSvgIcon(
+  <path
+    clip-rule="evenodd"
```
Comment (hcafaq) [inline packages/icons/src/paths/Others.tsx:7]: you'll get console error for this clip-rule to clipRule and fill-rule to fillRule.

Comment (Hey-Komal) [reply packages/icons/src/paths/Others.tsx]: Done

**Review by hcafaq — APPROVED** [review]
other than screenshot issues. code looks fine and works for me 

image

**Review by esraltintas — APPROVED** [review]

**Pull request merged**